std/conv.d

```diff
@@ -2180,6 +2180,36 @@ Lerr:
 
 @safe pure unittest
 {
+    void checkErrMsg(string input, dchar charInMsg, dchar charNotInMsg)
+    {
+        try
+        {
+            int x = input.to!int();
+            assert(false, "Invalid conversion did not throw");
+        }
+        catch(ConvException e)
+        {
+            // Ensure error message contains failing character, not the character
+            // beyond.
+            import std.algorithm.searching : canFind;
+            assert( e.msg.canFind(charInMsg) &&
+                   !e.msg.canFind(charNotInMsg));
```

Comment:
This is probably not the cleanest way to verify this bugfix, but I couldn't think of a better way, since `to!int` refuses to recognize a range that wraps around a string (it either fails or goes to a different overload that doesn't concern this fix), and strings are by-value ranges so there's no way to check the position of the range upon error.